playground/infrastructure/checker.py

```diff
@@ -90,10 +91,10 @@ def check_sdk_examples(paths: List[PurePath], sdk: Sdk, root_dir: str) -> bool:
 def main():
     args = parse_args()
 
-    root_dir = os.getenv("BEAM_ROOT_DIR")
+    root_dir = os.getenv(BEAM_ROOT_DIR_ENV_VAR_KEY)
     if root_dir is None:
         raise KeyError("BEAM_ROOT_DIR environment variable should be specified in os")
```

Comment:
Please use value from the new constant when building the error message so won't get out of sync with the actually expected value